test/testdrive/kafka-exactly-once-sink.td

```diff
@@ -289,7 +289,7 @@ $ kafka-verify format=json sink=materialize.public.json_avro sort-messages=true
   INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'json-reuse-topic'
     WITH (reuse_topic=true)
   FORMAT JSON
-JSON consistency topic not yet supported
+with FORMAT JSON, you need to manually specify a consistency topic using 'CONSISTENCY TOPIC consistency_topic CONSISTENCY FORMAT format'
 
```

Comment:
Isn't this the case where AVRO is the only acceptable format? So we should say that instead of the generic "CONSISTENCY FORMAT format".

Reply:
True :ok_hand: